h2/src/test/org/h2/test/scripts/functions/numeric/bitnot.sql

```diff
@@ -0,0 +1,16 @@
+-- Copyright 2004-2019 H2 Group. Multiple-Licensed under the MPL 2.0,
+-- and the EPL 1.0 (https://h2database.com/html/license.html).
+-- Initial Developer: Joe Littlejohn
+--
+
+create memory table test(id int primary key, name varchar(255));
+> ok
+
+insert into test values(1, 'Hello');
+> update count: 1
+
+select bitnot(null) vn, bitnot(0) v1, bitnot(10) v2, bitnot(-10) v3 from test;
+> VN   V1                   V2                   V3
+> ---- -------------------- -------------------- ------------
+> null 18446744073709551615 18446744073709551605 9
```

Comment:
`BIGINT` is a 64-bit signed integer in H2. H2, like many other databases, doesn't have unsigned data types. Expected output is `null -1 -11 9`. Launch `TestScript` by yourself and adjust your test case as necessary.